README.md

```diff
@@ -8,130 +8,478 @@ CRAN Task View: High-Performance and Parallel Computing with R
 | **Version:**    | 2018-01-05                                                   
```

Comment:
This is all auto-generated by `ctv`.

Reply:
Not quite. See the `ctv2md.r` script? It will create the README.md from the updated ctv.

But thanks for the PR. Will fold.  (Keeping lines at 80 or 100 or 120 columns is nice.  The diff viewer here at GH doesn't work to well with your long non-broken lines.  No worries now, but keep in mind for next time.)

And the diff below is just plain wrong.  It should not have change unaffected lines.

Reply:
See here:

```sh
edd@rob:~/git/ctv-hpc(master)$ git checkout -b pr14
Switched to a new branch 'pr14'
edd@rob:~/git/ctv-hpc(pr14)$ git pull https://github.com/terrytangyuan/ctv-hpc.git master
remote: Counting objects: 4, done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 4 (delta 2), reused 4 (delta 2), pack-reused 0
Unpacking objects: 100% (4/4), done.
From https://github.com/terrytangyuan/ctv-hpc
 * branch            master     -> FETCH_HEAD
Updating 3910ce6..464819a
Fast-forward
 HighPerformanceComputing.ctv |  14 ++++-
 README.md                    | 602 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++---------------------------------
 2 files changed, 502 insertions(+), 114 deletions(-)
edd@rob:~/git/ctv-hpc(pr14)$ 
```

602 lines?  I don't think so.  You changed 14 lines in the ctv.

I'll fix.

Reply:
Interesting. I literally just ran that R script and it updated the README.md for me. I think the version I installed from CRAN shortens each line it produces (probably different than the version of ctv package you are using).